### ISSUE TITLE
[Feature Request] The Ability To Separate and Sort Bookmarked Threads By Imageboard & Disable Notifications Automatically From Specific Boards

### ISSUE THREAD

**Issue by seniorm0ment** [Adamantcheese/Kuroba#1106]
Suggesting the ability to sort bookmarked/watched threads by imageboard instead of clustering them together. The ability to sort them by recently bookmarked, board, thread name, bump activity, etc. and the ability to seperate them by imageboard.

An idea for the latter would be a simple horizontal rule with the imageboard icon and text ontop of the horizontal rule.
Something like the following (but the text sitting right on the horizontal line for condensing purposes.

```
_____________________
logo 4chan
_____________________
4chan thread 1
4chan thread 2
____________________
logo Lainchan
____________________
lainchan thread 1
lainchan thread 2
```

and my other request is the ability to by default, set certain boards to automatically have notifications disabled.
For example, if one wishes to by default have any bookmarked threads from /h/ not show notifications, but always send notifications for /g/ they can set that. Instead of needing to manually enable or disable each thread's notifications it is just a default rule.

**Comment by Adamantcheese** [comment]
Duplicate of #178, but I guess I'll have another look at it.
As for the notifications thing, maybe I'll take a look at adding board settings, but don't count on it.

**Comment by Adamantcheese** [comment]
Per #1131, do you still want sorting options like the catalog, or is a search function sufficient? Totally understand if it isn't.

**Comment by seniorm0ment** [comment]
Yeah the sorting would be a lot easier than search, my main reason being it's nice to be able to see the watched threads from the different imageboards separated instead of all bunched together. Being able to search would be a bonus I guess, but I don't think I'd ever use seach personally. More looking for that board separation.

To be clear, I (personally) wouldn't want separation in a subfolder style, where I actually have to click on Lainchan or 4chan to bring up all the watched threads in there, but I'd prefer to have it jist as the list it already is, but just with dividers that say 4chan, Lainchan, etc with the icon.

Thanks, keep up the great work!
